plugins/modules/vmware_object_role_permission_info.py

```diff
@@ -26,6 +26,12 @@
     - "python >= 2.7"
     - PyVmomi
 options:
+  principal:
+    description:
+    - The optional name of an entity, such as a user, assigned permissions on an object.
+    - If provided, actual permissions on the specified object are returned for the principal, instead of roles.
+    type: str
+    required: False
```

Comment:
```suggestion
    version_added: '1.11.0'
```

Reply:
The entire module is being added in 1.11.0. Do I still need to add this?

Reply:
oh, I didn't realize. It is not required.